exercises/Perception-Driven_Manipulation/solution_ws/src/object_manipulation_msgs/srv/PlacePlanning.srv

```diff
@@ -4,7 +4,7 @@
 string arm_name
 
 # the object to be placed
-manipulation_msgs/GraspableObject target
+moveit_msgs/GraspableObject target
```

Comment:
It does not appear that the `GraspableObject` message actually exists in the `moveit_msgs` package for ROS Melodic. Were you able to successfully build the solution workspace in this exercise?

It also seems like the only message type from `object_manipulation_msgs` that gets used in this exercise is the `GraspHandPostureExecution` action. Since that is the case, I think we should just comment out the `manipulation_msgs/GraspableObject` lines